exercises/triangle/src/main/java/Triangle.java

```diff
@@ -4,7 +4,15 @@
         throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
     }
 
-    TriangleKind getKind() {
+    public boolean isEquilateral() {
```

Comment:
A very minor point (sorry to be picky) but you don't need `public` here. It's best practice to be as restrictive as possible with access modifiers and you only need this to be `package private` (which, as you might know, is the default so you don't have to specify it). Especially since the constructor and class are both `package private`, it makes sense for these methods to be as well :)